scripts/autopush.sh

```diff
@@ -42,5 +41,13 @@ for src in $(gsutil ls gs://datcom-control/autopush/*_latest_base_cache_version.
 done
 $ROOT/scripts/deploy_gke.sh autopush us-central1
 $ROOT/scripts/deploy_gke.sh autopush europe-west2
-# $ROOT/scripts/deploy_gke.sh feedingamerica us-central1
-$ROOT/scripts/deploy_gke.sh stanford us-central1
+
+# Deploy stanford instance
+# yq eval -i 'del(.tables)' deploy/overlays/stanford/custom_bigtable_info.yaml
+# yq eval -i '.tables = []' deploy/overlays/stanford/custom_bigtable_info.yaml
+# for src in $(gsutil ls gs://datcom-stanford-resources/control/latest_base_cache_version.txt); do
+#   echo "Copying $src"
+#   export TABLE="$(gsutil cat "$src")"
+#   yq eval -i '.tables += [env(TABLE)]' deploy/overlays/stanford/custom_bigtable_info.yaml
+# done
```

Comment:
for future, it might be nice to put this logic in a function used from main autopush and other custom DCs